internal/repos/metrics.go

```diff
@@ -234,10 +234,14 @@ select round((select cast(count(*) as float) from latest_state where state = 'er
 
 	backoffQuery := `
 -- source: internal/repos/metrics.go:src_repoupdater_errored_sync_jobs_total
-SELECT extract(epoch from max(now() - last_sync_at)) FROM external_services
+SELECT extract(epoch from max(now() - last_sync_at))
+FROM external_services as es
```

Comment:
The smallest of tiniest of nits. :stuck_out_tongue_closed_eyes: 

```suggestion
FROM external_services AS es
```